packages/tools/src/tools/segmentation/CircleROIStartEndThresholdTool.ts

```diff
@@ -353,12 +341,14 @@ class CircleROIStartEndThresholdTool extends CircleROITool {
           startCoord,
           viewplaneNormal
         );
+        data.startCoordinate = startCoord;
       }
       if (Array.isArray(endCoordinate)) {
         endCoord = this._getCoordinateForViewplaneNormal(
           endCoord,
           viewplaneNormal
         );
+        data.endCoordinate = endCoord;
       }
```

Comment:
Can we stick to `startCoordinate` and `endCoordinate` perhaps? I don't like the direction this is going, with `startCoord` and `startCoordinate`, etc.